modules/core/pom.xml

```diff
@@ -160,6 +160,9 @@
                             <value>org.apache.xerces.jaxp.validation.XMLSchemaFactory</value>
                         </property>
                     </systemProperties>
+                    <environmentVariables>
+                        <SOAP_SERVICE_TEST>http://localhost:9000/services/SimpleStockQuoteService</SOAP_SERVICE_TEST>
```

Comment:
Is this system variable getting set only when we are running tests? If not can we limit the scope to tests?

Reply:
Resolved https://github.com/apache/synapse/pull/55/commits/9d048ed5b9546a1e670131114c84df3a6e67ad50